styleguide.config.js

```diff
@@ -23,6 +23,10 @@ module.exports = {
         {
           name: 'TextLockups',
           components: 'src/molecules/text-lockups/**/*.js'
+        },
+        {
+          name: 'StepProgress',
```

Comment:
I suggest making an "Other" or "Misc" directory, so that we don't have to add a new section to the config file for every molecule.